nextflow.config

```diff
@@ -20,7 +20,7 @@ params {
   save_reference = false
   min_aln_length = 15
   star_ignore_sjdbgtf = false
-  seq_center = false
+  seq_center = ''
```

Comment:
Can we revert these changes now?

Reply:
Not sure yet, the Schema validator doesn't like it either. And it makes sense, you expect a String then it can't be a boolean.
Might be a way to workaround it - have to get warm with the library first :)